roles/build-release/defaults/main.yaml

```diff
@@ -1,4 +1,7 @@
 ---
+# The command to run `antsibull-build`
+antsibull_build_command: poetry run antsibull-build
```

Comment:
I'm trying to picture if there is a use case for providing the flexibility of specifying an arbitrary command... In other words, would the command ever be something other than either:
- ``poetry run antsibull-build`` or
- ``poetry run coverage run -p --source antsibull -m antsibull.cli.antsibull_build`` ?

I never got around to finishing https://github.com/ansible-community/antsibull/pull/304 but my train of thought was that it would be a toggle to enable coverage, not a way to specify a different command.

Something like ``antsibull_enable_coverage`` that would default to ``false`` and when ``true`` would take care of flipping the command.

Maybe something along the lines of:
```
- name: Set build command
  vars:
    _with_coverage: "poetry run coverage run -p --source antsibull -m antsibull.cli.antsibull_build"
    _without_coverage: "poetry run antsibull-build"
  set_fact:
    _build_command: "{{ antsibull_enable_coverage | ternary(_with_coverage, _without_coverage) }}"

# [...]

- name: Prepare a release with new dependencies
  command: >-
    {{ _build_command }} prepare {{ antsibull_ansible_version }}
    {{ antsibull_build_command }} prepare {{ antsibull_ansible_version }}
      --data-dir {{ antsibull_data_dir }}
      {{ _feature_freeze | default('') }}

# [...]

```

What do you think ?

Reply:
As discussed on IRC, I think it's better to have a generic way to specify how `antsibull-build` is called (with/without poetry, with/without coverage, something completely different?).